Dockerfile

```diff
@@ -1,7 +1,9 @@
 ARG BUILD_STAGE_VERSION=v0.2.0
 ARG DEBIAN_VERSION=11.7
 
-FROM ghcr.io/cnpem/lnls-debian-11-epics-7:${BUILD_STAGE_VERSION} AS build-stage
+FROM ghcr.io/cnpem/lnls-debian-11-epics-7:${BUILD_STAGE_VERSION} AS build-image
+
+FROM build-image as build-stage
```

Comment:
`AS` seems to be the standard for our `FROM` directives.

Reply:
Good catch. My mistake.